### PR TITLE
Log in as user

### PR DESCRIPTION
**Pull request by dftd**

### CODE REVIEW

File: oc-includes/osclass/classes/controller/admin/CAdminUsers.php [hunk 1]
```diff
@@ -317,6 +320,27 @@ public function doModel()
                 osc_add_flash_ok_message($msg, 'admin');
                 $this->redirectTo(osc_admin_base_url(true) . '?page=users');
                 break;
+            case ('login'):
```
Comment (navjottomer) [inline oc-includes/osclass/classes/controller/admin/CAdminUsers.php:323]: login case is too generic make it more unique so it wouldn't confuse anyone.

Comment (dftd) [reply oc-includes/osclass/classes/controller/admin/CAdminUsers.php]: @navjottomer done

**Comment by navjottomer** [comment]
Remove bender changes fro PR

**Comment by dftd** [comment]
@navjottomer aaah, how? Do I need to make a new PR?

**Pull request closed without merging**